.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-- '5'
+- '6.10.3'
```

Comment:
unit tests fail with node 5. Also, there were a lot of improvements in node 6, and in general we should endeavor to stay as up to date as possible with our packages.